packages/driver/cypress/e2e/cypress/request-logs.cy.ts

```diff
@@ -0,0 +1,443 @@
+import { expect } from 'chai'
+
+describe('Request Logs', () => {
```

Comment:
A lot of this file is the same as it was in `proxy-logging.cy.ts`, but GitHub's diff viewer doesn't seem to detect the rename.